### PR TITLE
signup/login page done

### PR DESCRIPTION
**Pull request by TheDivineAurora**
Issue: #1
made a component SignUp in components folder and imported it in home page to display the content !

### CODE REVIEW

**Comment by VatsalBhuva11** [comment]
@TheDivineAurora please attach a screenshot/video of the changes that you've made.

File: src/components/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+
+const SignUp = () => {
```
Comment (bhupesh98) [inline src/components/SignUp.js:3]: Use react hooks to save the form data and a http request to send it to backend, make the link as a environment variable for now, refer to this while creating env variables https://create-react-app.dev/docs/adding-custom-environment-variables/

**Review by bhupesh98 — CHANGES_REQUESTED** [review]
Do send the short video, with you filling the form and the submit button is functioning (just make a console log and preferrably use axios for post request. 

.env.example must be updated too when you edit your .env file

File: src/components/SignUp.js (hunk 1 above)
Comment (VatsalBhuva11) [reply src/components/SignUp.js:3]: the backend will be available on localhost:4000/api/v1
the user registration controller available on /api/v1/auth/register
make a POST request to the above route, sending the username, email, and the password as the form data.

**Comment by bhupesh98** [comment]
@TheDivineAurora updates??

**Comment by TheDivineAurora** [comment]
Give me some time I will complete it

**Comment by VatsalBhuva11** [comment]
@TheDivineAurora yeah it's cool. make sure the register button is functioning at the least, as the register functionality has been done on the backend. to test it, ping me on discord (@vatsal) in GrepIt-Backend, or personally

**Comment by VatsalBhuva11** [comment]
@TheDivineAurora since you closed this PR, will you be opening a new PR, or shall we assign it someone else?

**Pull request closed without merging**